src/mixpanel-core.js

```diff
@@ -2762,7 +2763,11 @@ MPNotif.prototype._attach_and_animate = _.safewrap(function() {
             self.dismiss();
             if (self.clickthrough) {
                 self._track_event('$campaign_open', {'$resource_type': 'link'}, function() {
-                    window.location.href = self.dest_url;
+                    if(self.mixpanel.get_config('message_link_new_window')){
+                        window.open(self.dest_url);
```

Comment:
Since you're not unloading the current page here, it's probably a slightly better experience to move this outside the `track()` callback, so the new window/tab opens immediately rather than after the network call or timeout. I'm currently adding this branch to the next release and will make that change unless you have strong objections @mkdai.

Reply:
Sounds good, no objections. Thanks a lot